docs/api/websockets.md

```diff
@@ -173,42 +175,59 @@ Bun.serve<WebSocketData>({
 });
 ```
 
+To connect to this server from the browser, create a new `WebSocket`.
+
+```ts#browser.js
+const socket = new WebSocket("ws://localhost:3000/chat");
+
+socket.addEventListener("message", event => {
+  console.log(event.data);
+})
+```
+
+The cookies that are currently set on the page will be sent with the WebSocket upgrade request and available on `req.headers` in the `fetch` handler. Parse these cookies to determine the identity of the connecting user and set the value of `data` accordingly.
```

Comment:
Maybe add a section header here because its very commonly asked and people might miss this

### Identifying Users